main.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+    <title>Communet</title>
+    <link rel="stylesheet" href="style.css"> 
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"/>
+</head>
+<body>
+    <!-- home section start -->
+    <header class="home">
+        <h1>Commune</h1>
+    </header>
+    <!--about us page-->
+
+    <section class="about-us">
+        <div class="ab-container">
+            <div class="row">
+                <div class="col-sm-5">
+                    <h1 class="text-center about-us-h">About us</h1>
+                    <p class="about-us-p">The Communet platform allows people with shared interests and values to create a financially cooperative community, with pre-established norms and procedures.
+
+Chores, shopping, and bills are all pre-configured and agreed upon before co-habitation begins, taking the mess and complexity out of starting a shared life.
+
+Future chores, expenses, and savings can be added and voted on at any time.
+                    </p>
+                </div>
+                <div class="col-sm-5">
+                    <div class="img-class" alt="">
+                        <img src="images/cm2.png" alt="">
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+
+    <!--our team page-->
+    <main class="container">
+        <div class="header">
+            <h1>Our Team</h1>
+        </div>
+        <div class="sub-container">
+            <div class="teams">
```

Comment:
div soup. this should be an `<ol>` or a `<ul>` with css adjustments

consider using opengraph attributes as well for the individual list items